visualize.py

```diff
@@ -3,15 +3,14 @@
 import moviepy.editor as mpy
 import random
 import torch
-from scipy.misc import toimage
+import scipy.misc
```

Comment:
Maybe we can change this to import only the required functions from scipy.misc since only a few lines of code require a function from this package

Reply:
Done!